doc/design/communications.md

```diff
@@ -0,0 +1,92 @@
+# Communications
+
+This document discusses the communication channels that are possible between components, but not what flows over them. This is essentially Layer2/3 and Layer7 is discussed in the documents relating to specific components that talk over these channels.
```

Comment:
Change : the communication channels that are possible between components
to: the communication channels supported between components